### ISSUE TITLE
Middle click on macOS

### ISSUE THREAD

**Issue by ris58h** [ngdeleito/easyGestures#16]
Hi. I've added the extension to Firefox on macOS Monterey and `shift+left_click` works but middle click doesn't. I use https://github.com/artginzburg/MiddleClick-BigSur to emulate middle click behavior on macOS. It works fine (at least it opens a link in a new tab in Firefox).

**Comment by ngdeleito** [comment]
Hi, thank you for your message. Do I understand correctly that (1) you use MiddleClick to emulate middle click on macOS, and (2) you cannot open the pie menu using the middle click button?

I'm afraid MiddleClick might be simply emulating the middle click behavior somehow, but not reporting to the OS a middle click event (or Firefox not reporting a middle click event to the extension)? I used to use MagicPrefs, but, even though it keeps working on the latest macOS, the OS no longer provides the APIs for it to fully work... Since then, I have moved to using classic mice with a physical middle click button.
